middle_end/flambda2.0/from_lambda/lambda_to_flambda_primitives.ml

```diff
@@ -305,10 +308,10 @@ let convert_lprim ~backend (prim : L.primitive) (args : Simple.t list)
     tag_int (Binary (String_or_bigstring_load (Bytes, Eight), arg1, arg2))
   | Pbytesrefs, [arg1; arg2] ->
     let module B = (val backend : Flambda2_backend_intf.S) in
-    string_or_bytes_ref B.size_int Bytes arg1 arg2 dbg
+    string_or_bytes_ref ~size_int:B.size_int Bytes arg1 arg2 dbg
```

Comment:
I would let-bind `size_int` further up to `B.size_int`, then you can just write `~size_int` here, and the same below.

Reply:
(And of course hoist the unpacking of `backend` to `B`.)